lib/addons/karpenter/index.ts

```diff
@@ -396,20 +395,19 @@ export class KarpenterAddOn extends HelmAddOn {
             path: '/'
         });
         
-        const clusterName = cluster.clusterName;
-        const clusterHash = md5.Md5.hashStr(clusterName);
+        const clusterId = Names.uniqueId(cluster);
```

Comment:
isn't that `six of one, half a dozen of the other`? cluster seems to be fairly dynamic and specific to the env, like actual build box, time, etc. 
A simple composite identifier like `uniqueId('${blueprintProps.name}-${region}'`  maybe a more straighforward choice.